include/swift/Runtime/HeapObject.h

```diff
@@ -84,8 +84,12 @@ struct TwoWordPair {
   // in registers, so cram the result into an unsigned long long.
   // Use an enum class with implicit conversions so we don't dirty C callers
   // too much.
-#if __arm__ || __i386__
+#if __arm__ || __i386__ || defined(__CYGWIN__)
+#if defined(__CYGWIN__)
+  using Return = unsigned __int128;
```

Comment:
I encountered an error when I first tried same as yours, so committed with 'using' keyword. Just, I retried it again, no error occurred at this time. It works fine now. Perhaps I missed something when I commit.
I'll replace the 'using' to 'enum' syntax, thank you.